assets/airports/rjss.json

```diff
@@ -0,0 +1,960 @@
+        "Y142": ["GTC", "NESKO", "SAMON"],
+        "Y15" : ["TENRU", "MYOGI", "TADMI", "DANDY", "HEIKE", "INAWA", "SHIRO", "NIRAX", "OWLET"],
+        "Y303": ["GTC", "VIKET", "AKTIL", "SASAP", "ESKAS", "ENTAK"],
+        "Y312": ["KCC", "AZUMI", "PAPNA", "NAEBA", "MAGNA", "GTC", "KENSI", "YAYOI", "UWE"],
+        "Y347": ["IGROD", "EKVIK", "HANAB", "GOLDO", "GTC"],
+        "Y45":  ["GTC", "TERAD", "HAKBA", "GORYU", "IKUJI", "URUSI", "IMIZU", "OYABE", "KMC", "SONBU", "MADUL", "SAKYU", "MIHOU", "KYOKA", "CARPS", "MARCO", "SQRUM", "YANAI", "OOITA", "SOBOH", "RAFUL", "MOCOS", "HKC", "ISKID", "MAKRA", "SAZMA", "BOMAP", "ANOXA", "MEKAX", "HACHA", "ONC", "LEXUS", "OKUMA", "NHC"],
+        "Y512": ["ADNAP", "OATIS", "BEKEN", "SDE", "ELDAK", "GTC"],
+        "Y514": ["ASTER", "SDE"],
+        "Y515": ["KMC", "ZUWAI", "ETIGO", "VIKET", "AIZUH", "LANCE"],
+        "Z119": ["GTC", "YANAT", "FKE"]
+    },
+    "sids": {
+        "CUBIC3": {
+            "icao": "CUBIC3",
+            "name": "Cubic Three",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "SS901"],
+                "RJSS27": [["_RWY0902DME", "A5+"], "BUBLE"]
+            },
+            "body": [["CUBIC", "A150+"]],
+            "exitPoints": {
+                "SAMBO": ["RIDER", "SAITI", "SAMBO"]
+            },
+            "draw": [
+                ["SS901", "CUBIC"],
+                ["BUBLE", "CUBIC"],
+                ["CUBIC", "RIDER", "SAITI", "SAMBO*"]
+            ]
+        },
+        "DERBY3": {
+            "icao": "DERBY3",
+            "name": "Derby Three",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "SS901", "ANEMO", "EBOSI"],
+                "RJSS27": [["_RWY0901DME", "A5+"], "SS701", ["_SS701233001", "A13+"], "EBOSI"]
+            },
+            "body": ["DERBY"],
+            "exitPoints": {
+                "GTC": ["GTC"]
+            },
+            "draw": [
+                ["SS901", "ANEMO", "EBOSI"],
+                ["SS701", "EBOSI"],
+                ["EBOSI", "DERBY", "GTC*"]
+            ]
+        },
+        "IXE8": {
+            "icao": "IXE8",
+            "name": "Iwaki Eight",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "^_RWY2703DME", "_SDE120005"],
+                "RJSS12": [],
+                "RJSS27": ["^_SDE120005"],
+                "RJSS30": ["_SDE120005"]
+            },
+            "body": ["_IXE024SDE120", ["_IXE024047", "A110+"], ["_IXE024028", "A150+"]],
+            "exitPoints": {
+                "IXE": ["IXE"]
+            },
+            "draw": [
+                ["_SDE120005", "_IXE024SDE120", "IXE"]
+            ]
+        },
+        "SDE6R": {
+            "icao": "SDE6R",
+            "name": "Sendai Reversal Six",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "^_RWY2703DME", "_SDE120005", "^_SDE120010"],
+                "RJSS12": ["^_SDE120010"],
```

Comment:
Could be good to add `_SDE120005` in here as well to ensure aircraft fly the radial